contrib/test_mad.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -exE -o pipefail
+
+export HCA="mlx5_0:1"
```

Comment:
can we detect first active port instead of hardcoded?

Reply:
Done, see `detect_hca()` in the Shell script.